### PR TITLE
docs(django): enabling automatically and manually

### PR DESCRIPTION
**Pull request by majorgreys**

### CODE REVIEW

File: ddtrace/contrib/django/__init__.py [hunk 1]
```diff
@@ -191,8 +205,9 @@
    tracer.set_tags({'env': 'production'})
 
    import my.custom.tracer
-   from ddtrace import Pin
+   from ddtrace import Pin, patch_all
    import django
+   patch_all()
```
Comment (brettlangdon) [inline ddtrace/contrib/django/__init__.py]: is `django` enabled by default now? does this need `patch_all(django=True)`?

File: ddtrace/contrib/django/__init__.py [hunk 2]
```diff
@@ -104,6 +115,9 @@
 3. Remove ``TraceMiddleware`` or ``TraceExceptionMiddleware`` if used in
    ``settings.py``.
 
+3. Enable Django tracing automatically via `ddtrace-run`` or manually by
+   adding ``patch_all()`` to ``settings.py``.
```
Comment (brettlangdon) [inline ddtrace/contrib/django/__init__.py]:
```suggestion
   adding ``ddtrace.patch_all()`` to ``settings.py``.
```

?

Comment (brettlangdon) [reply ddtrace/contrib/django/__init__.py]: no strong opinion, just a thought

File: ddtrace/contrib/django/__init__.py (hunk 1 above)
Comment (majorgreys) [reply ddtrace/contrib/django/__init__.py]: Yes it's enabled by default so left it out to avoid redundancy.

Comment (brettlangdon) [reply ddtrace/contrib/django/__init__.py]: perfect

**Pull request merged**